cloudinit/config/cc_growpart.py

```diff
@@ -139,25 +142,46 @@ def available(self):
             pass
         return False
 
+    def get_size(self, output):
+        """
+        CHANGED: resizer=sfdisk partition=1 start=2048 old: size=1021952
+                 end=1024000 new: size=2045919 end=2047967
+        CHANGED: partition=1 start=2048 old: size=1021952 end=1024000
+                 new: size=2045919,end=2047967
```

Comment:
This 'size' is in sectors. (I'm 95% sure of that)
Previously get_size() returned size in bytes.

So you have to figure out what the sector size is and multiply that.

Thats what I suggested adding to growpart.

Reply:
Good catch, I'll confirm and convert to bytes